src/main/kotlin/me/hubertus248/deployer/applications/exception/ApplicationCorruptedException.kt

```diff
@@ -0,0 +1,6 @@
+package me.hubertus248.deployer.applications.exception
+
+import me.hubertus248.deployer.common.exception.ApplicationException
+import org.springframework.http.HttpStatus
+
+class ApplicationCorruptedException(message: String) : ApplicationException(HttpStatus.INTERNAL_SERVER_ERROR, message)
```

Comment:
Some documentation would be nice (What does it mean? When is it thrown?)

Reply:
good idea